### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by BlasterLink**

### CODE REVIEW

**Comment by codeguru42** [comment]
If you flesh this out more, I will be glad to merge it.

**Comment by BlasterLink** [comment]
nah, i get it from both amazon and play store and sorry because my native
language is not English that can make me a bit hard to understand the
meaning of what you just read. but I get the point.

and thanks for suggesting me, that's make me want to contribute more on
open source.

Pada tanggal Rab, 9 Okt 2019 pukul 05.13 Code Apprentice <
notifications@github.com> menulis:

**Comment by BlasterLink** [comment]
nah, i get it from both amazon and play store and sorry because my native
language is not English that can make me a bit hard to understand the
meaning of what you just write. but I get the point.

and thanks for suggesting me, that's make me want to contribute more on
open source.

Pada tanggal Rab, 9 Okt 2019 pukul 05.13 Code Apprentice <
notifications@github.com> menulis:

**Comment by codeguru42** [comment]
At the top of this page is a tab that says "Wiki". It has several other pages that have content that might be useful in the README.

Don't worry about your English. Do your best and keep learning. Don't apologize. I will ask questions when I don't understand.

**Comment by codeguru42** [comment]
@BlasterLink Will you have time to make the requested changes this week?

**Comment by BlasterLink** [comment]
maybe at the end of this night

Pada tanggal Min, 13 Okt 2019 pukul 02.27 Code Apprentice <
notifications@github.com> menulis:

**Comment by BlasterLink** [comment]
I'm Already Change the readme and add some link from wiki, could you check
that and correct me if i were wrong

Pada tanggal Min, 13 Okt 2019 pukul 02.27 Code Apprentice <
notifications@github.com> menulis:

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+**What's new in version 0.6.8.1**
+* Version 0.6.8.1
+* Fix bug that prevents editing cards
+
+Contributing
+-
+
+If you are interested in contributing to BBCT, please join our [Google Group](https://groups.google.com/forum/?hl=en#!forum/bbct). Then [download and install the desktop app](https://sourceforge.net/projects/bbct/files/latest/download) and/or the Android app from Google Play. Read [Getting Started](https://github.com/BaseballCardTracker/bbct/wiki/Getting-Started) for instructions about what tools to install and how to get the code. Finally, browse the open issues to find a task that interests you. When you are ready to submit changes, make a [Pull Request](https://help.github.com/en/articles/about-pull-requests) for a code review. If you have any questions, please [post the our Google Group](https://groups.google.com/forum/?hl=en#!newtopic/bbct), Make sure you already join the Group before asking a question.
+
+**Here is some link to help contribute**
```
Comment (codeguru42) [inline README.md]: Put "Additional Links" for this title.

Comment (codeguru42) [reply README.md]: I see that you marked this comment as "resolved", but I don't see the requested change. Please let me know if you have any questions.

**Comment by BlasterLink** [comment]
that is the last change before you comment to put additional link on title
and I don't understand the meaning of tittle that you mentioned on the last
email. can you give me more details about this "tittle" ?


Pada tanggal Sel, 29 Okt 2019 pukul 06.51 Code Apprentice <
notifications@github.com> menulis:

**Comment by codeguru42** [comment]
I mean that I want you to change the wording of the line where I put the comment. Change the words "Here is some link to help contribute" to "Additional Links".

**Comment by BlasterLink** [comment]
I Changed it

>

**Pull request merged**